### ISSUE TITLE
Where's the code?

### ISSUE THREAD

**Issue by coingraham** [MitocGroup/terraform-aws-control-tower#3]

**Comment by eistrati** [comment]
@coingraham Sorry for the delay in our response. This code is still NOT ready to be published. But it's a priority for us and we anticipate to have it ready for prime time in the next couple of weeks.

P. S. I hope they are treating you well at Slalom ;)

**Comment by coingraham** [comment]
It's going well.  This came up in conversation and I posted a link to your repo only to find later that it's empty.  I think this is something people are looking for.

**Comment by sondraschrodervector** [comment]
Yes, we are definitely looking for it. Please let us know how we can help
